### PR TITLE
BigEarthNet Splits

### PR DESCRIPTION
**Pull request by isaaccorley**
- Adds official train/val/test splits to BigEarthNet
- Adds option to use original 43 classes or updated 19 classes
- Updates configs/tests/datamodules

### CODE REVIEW

**Review by calebrob6 — DISMISSED** [review]
The trainer is currently hardcoded with 43 classes, can probably sort that out later

**Comment by isaaccorley** [comment]
> The trainer is currently hardcoded with 43 classes, can probably sort that out later

Ahh yeah I need to make another PR to modify the tasks.

File: tests/datasets/test_bigearthnet.py [hunk 1]
```diff
@@ -21,7 +22,9 @@ def download_url(url: str, root: str, *args: str, **kwargs: str) -> None:
 
 
 class TestBigEarthNet:
-    @pytest.fixture(params=["all", "s1", "s2"])
+    @pytest.fixture(
+        params=product(["all", "s1", "s2"], [43, 19], ["train", "val", "test"])
```
Comment (adamjstewart) [inline tests/datasets/test_bigearthnet.py:26]: Let's use `zip` instead of product to reduce the total testing time needed

**Review by adamjstewart — COMMENTED** [review]
One small change and this is ready to go!

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**